### ISSUE TITLE
PREMIS events show the version of the plugin and not the version of the tool

### ISSUE THREAD

**Issue by jmaferreira** [keeps/roda#291]
This is wrong. The Anti virus task says its version 1.0. Thats the version of the plugin, but we need to know the version of ClamAV in the event.

Have it hardcoded (making sure that the correct version of the tool is installed) or extract version from the tool automatically.

**Comment by luis100** [comment]
Use the following:
```
$ clamscan --version
ClamAV 0.98.7
```

Or in the case of the server mode:
```
$ clamdscan --version
ClamAV 0.98.7/21486/Tue Apr  5 22:19:10 2016
```

**Comment by hsilva-keep** [comment]
Solved via 72afb29ca543645a6a55f280eecf50cd36dc548e
